client/src/test/java/org/spine3/base/IdentifierShould.java

```diff
@@ -40,18 +40,18 @@
 
     @Test
     public void getDefaultValue_by_class_id() {
-        assertEquals(0L, Identifier.getDefaultValue(Long.class).longValue());
-        assertEquals(0, Identifier.getDefaultValue(Integer.class).intValue());
-        assertEquals("", Identifier.getDefaultValue(String.class));
-        assertEquals(Timestamp.getDefaultInstance(), Identifier.getDefaultValue(Timestamp.class));
+        assertEquals(0L, getDefaultValue(Long.class).longValue());
```

Comment:
I'd have separate tests for each type. Right now, if something is broken, it's more difficult to trace the reason.

Same applies to the test method below.